stestr/commands/run.py

```diff
@@ -429,6 +454,29 @@ def run_tests():
                 if result:
                     return result
 
+    if pdb:
+        ids = pdb
+        if '::' in ids:
+            ids = ids.replace('::', '.')
+        if ids.find('/') != -1:
```

Comment:
'/' can be '\' in Windows case? But this is a different topic.

Reply:
Oh, that's a good point.  I just copied this section from `- - no-discover`'s block. Yeah I agree we should probably refractor this section in a follow up pr.